t/app/controller/waste_kands_bulky.t

```diff
@@ -529,7 +532,8 @@ FixMyStreet::override_config {
             # to Echo
             $mech->content_lacks('/waste/12345/bulky/cancel');
             $mech->content_lacks('Cancel this booking');
-
+            FixMyStreet::Script::Alerts::send_updates();
+            is $mech->email_count_is(0), 1, 'Update cancelled as would go to body user not report user';
```

Comment:
So I thought this test was checking that the cancel update wasn't being sent, but the test still works even removing the suppress the cancel update - removing it as pointless.